### PR TITLE
[ Unit Test ] Conv2D Forwarding Unit Test @open sesame 06/12 13:41

### PR DESCRIPTION
**Pull request by jijoongmoon**
This PR provides Conv2D Forwarding Unit Test.
. update python script to generate random input / kernel / golden Data
. update unit test code for conv2d & evaluate results

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: test/input_gen/genInput.py [hunk 1]
```diff
@@ -10,22 +10,107 @@
...
+##
+# @brief save data into file with filename
+# @param[in] data The data to be saved
+def save(filename,data):
+    if os.path.isfile(filename):
+        os.remove(filename)
+
+    with open(filename, 'ab') as outfile:
+        np.array(data, dtype=np.float32).tofile(outfile)
+    print( data.shape, " data is generated")
```
Comment (dongju-chae) [inline test/input_gen/genInput.py:27]: Minor comment: unnecessary space

File: test/input_gen/genInput.py [hunk 2]
```diff
@@ -10,22 +10,107 @@
...
     if os.path.isfile(outfile_name):
         os.remove(outfile_name)
 
-    for i in range(0,batch):
-        imarray=numpy.random.rand(channel, height, width)*255
-        with open(outfile_name, 'ab') as outfile:
-            numpy.array(imarray, dtype=numpy.float32).tofile(outfile)
-        print(i+1, " x ", imarray.shape, " data is generated")
-        print(imarray)
+    x=np.random.rand(batch,channel, height, width)
```
Comment (dongju-chae) [inline test/input_gen/genInput.py:42]: Minor comment: need some spaces between variables.

File: test/input_gen/genInput.py [hunk 3]
```diff
@@ -10,22 +10,107 @@
...
 
 if __name__ == "__main__":
-    if len(sys.argv) != 6:
-        print('wrong argument : filename, batch, channel, height, width')
-    else:
+    target = int(sys.argv[1])
+
+# Input File Generation with given info
+    if target == 1:
+        if len(sys.argv) == 7 :
```
Comment (dongju-chae) [inline test/input_gen/genInput.py:89]: Maybe `!=` ?

File: test/input_gen/genInput.py (hunk 1 above)
Comment (jijoongmoon) [reply test/input_gen/genInput.py]: fixed. Thanks.

File: test/input_gen/genInput.py (hunk 2 above)
Comment (jijoongmoon) [reply test/input_gen/genInput.py]: fixed. Thanks.

File: test/input_gen/genInput.py (hunk 3 above)
Comment (jijoongmoon) [reply test/input_gen/genInput.py]: fixed. Thanks.

**Pull request merged**